SUMMARY.md

```diff
@@ -178,6 +178,7 @@
   * [Azure Log Analytics](pipeline/outputs/azure.md)
   * [Azure Logs Ingestion API](pipeline/outputs/azure_logs_ingestion.md)
   * [Counter](pipeline/outputs/counter.md)
+  * [Dash0](pipeline/outputs/dash0.md)
```

Comment:
is not this one @patrick-stephens ?

Reply:
Weird, I checked a few times to confirm and missed it :)